src/main/java/frc/robot/commands/FollowPath.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.Swerve;
+import frc.robot.auto.Path;
+import frc.robot.auto.PathFollower;
+import frc.robot.auto.PathState;
+import frc.robot.libraries.PController;
+import frc.robot.subsystems.SwerveDrive;
+
+public class FollowPath extends CommandBase {
+
+  SwerveDrive m_drivetrain;
+
+  Path path;
+  PathFollower m_pathFollower;
+
+  // TODO Tune kP
+  PController rotationController = new PController(0.01, -1, 1);
```

Comment:
Make sure this can handle angles from 0 to 2pi, so a target of 1.9 pi and a current of .01pi will rotate the shorter direction. I think the gyro only returns a value from 0 to 2pi